pkg/utils/host/host.go

```diff
@@ -142,11 +131,17 @@ func AddFlags(command *cobra.Command) {
 		false,
 		"Automatically run in a privileged systemd unit if lacking enough capabilities",
 	)
+
+	// Enable the mount workaround by default when running inside a container.
+	automountFilesystemsDefault := false
+	if HostRoot != "" {
```

Comment:
```suggestion
	if HostRoot != "" && HostRoot != "/" {
```

See line 43 for explanations.